resources/Shaders/ExplosionEffect.geom.glsl

```diff
@@ -37,161 +42,128 @@ out VertexData{
 layout(triangles) in;
 layout(triangle_strip, max_vertices = 3) out;
 
+vec3 EqualTo(vec3 x, vec3 y) {
```

Comment:
Were vec3s for the sake of re-usability in case of updates. Changed to floats now.